in_toto/models/serialization.py

```diff
@@ -0,0 +1,64 @@
+# Copyright New York University and the in-toto contributors
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+<Program Name>
+  serialization.py
+
+<Started>
+  Sep 6, 2022
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Provides serialization/ deserialization classes
+  for various classes in the model.
+
+"""
+
+from typing import Union
+
+from securesystemslib.exceptions import DeserializationError
+from securesystemslib.serialization import JSONDeserializer
+
+from in_toto.exceptions import InvalidMetadata
+from in_toto.models.layout import Layout
+from in_toto.models.link import Link
+from in_toto.models.metadata import Envelope, Metablock, ENVELOPE_PAYLOAD_TYPE
+
+
+class PayloadDeserializer(JSONDeserializer):
+  """Deserialize JSON bytes into Link or Layout."""
```

Comment:
I think you can remove this docstring. The one in deserialize should be enough.

Reply:
Pylint shows warning.